pandas/tests/types/test_common.py

```diff
@@ -80,3 +81,22 @@ def test_dtype_equal_strict():
     assert not is_dtype_equal(
         pandas_dtype('datetime64[ns, US/Eastern]'),
         pandas_dtype('datetime64[ns, CET]'))
+
+
+get_dtype_funcs = [
+    (com.is_dtype_equal, 2),
```

Comment:
Well the whole point was to check all of the functions that `_get_dtype`.  The ones in the `list` are the only ones that do.